src/services/transfer/transfer-out-ehr-core.js

```diff
@@ -93,13 +94,30 @@ const isEhrRequestDuplicate = async conversationId => {
   return false;
 };
 
-const handleCoreTransferError = async (error, conversationId) => {
+const handleCoreTransferError = async (
+  error,
+  nhsNumber,
+  odsCode,
+  conversationId,
+  ehrRequestId
+) => {
   switch (true) {
+    // TODO PRMP-534 should a presigned URL not found be handled differently or is it considered an 06 patient not at practice?
     case error instanceof PresignedUrlNotFoundError:
+    case error instanceof PatientRecordNotFoundError:
+
+      await sendAcknowledgement(
+        nhsNumber,
+        odsCode,
+        conversationId,
+        ehrRequestId,
```

Comment:
The 4th argument of sendAcknowledgement() seems to be `messageId`, not sure if that would be the same as `ehrRequestId`.  If they are same then would be fine

Reply:
You've stumbled into something here, it might not be the correct ID to reference! `transferOutEhrCore` is being called with 5 arguments including a `messageId`, but the function only takes 4 parameters, not including a messageId. I've messaged you to discuss this on a call to see what we think the best course of action is.